documentation/src/pages/index.js

```diff
@@ -170,6 +170,15 @@ function Home() {
               Demos
             </Link>
           </div>
+          <Link
+            className={classnames(
+              'button button--link button--md text-white underline transition duration-700 ease-in-out hover:no-underline hover:text-green-100',
+              styles.getStarted,
+            )}
+            to={useBaseUrl('https://github.com/ivan-dalmet/formiz/issues/116')}
+          >
+            Formiz v2 is under construction !
+          </Link>
```

Comment:
1. I think that you can use a basic HTML `a` tag for this as `Link` is [made to navigate in Docusaurus](https://docusaurus.io/docs/docusaurus-core#link) and not outside.
2. I'm not sure about `useBaseUrl` as the name doesn't match what we want to do here.
3. Be careful to use `target="_blank"` to open the link in a new tab and be careful about its security.
4.. Thanks for the contribution.

Reply:
Thank you for your review ! Currently working on fixing what I've done.